R/scmake.R

```diff
@@ -50,13 +50,13 @@ scmake <- function(
   if(verbose) message('Starting build at ', start_time)
   out <- tryCatch({
     remake::make(
-    target_names=target_names, ..., verbose=verbose,
-    allow_missing_packages=allow_missing_packages, remake_file=remake_file)
+      target_names=target_names, ..., verbose=verbose,
+      allow_missing_packages=allow_missing_packages, remake_file=remake_file)
   }, error = function(e) {
     update_build_files(target_names = target_names, remake_file = remake_file, status_pre = status_pre)
-    stop(e$message)
+    stop(e)
```

Comment:
This was the change that caused the `loop_tasks()` error messages to start reporting the calling function correctly again